doc/source/user_guide/tutorial_parallelization_dask.ipynb

```diff
@@ -0,0 +1,364 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Batch processing with Dask\n",
+    "\n",
+    "By Mark Harfouche, Sept 2018\n",
```

Comment:
Usually, we don't put names and date here. If you want to specify some credits, you can fill up CONTRIBUTORS.txt

Reply:
thanks.